app/routes/api/campaign.js

```diff
@@ -8,27 +8,27 @@ router.route('/campaigns')
 
 // create new campaign
 .post((request, response) => {
-    if(request.user) {
-        campaignPromise = campaignController.saveCampaign(request);
-        campaignPromise.then(function() {
-            return campaignController.findCampaignByTitleAndUser(request.body.title, request.user.id);
-        })
-        .then(function(result) {
-            response.send(result[0]._id);
-        })
-        .catch(function(err) {console.log(err)});
+    if (request.user) {
+        campaignController.saveCampaign(request)
+            .then(function() {
+                return Campaign.find().byTitleAndUser(request.body.title, request.user.id);
+            })
+            .then(function(result) {
+                response.send(result[0]._id);
+            })
+            .catch(function(err) {console.log(err)});
     } else {
         response.status(301).render('unauthorized', {logged_in: false});
     }
 })
 
 // fetch campaign list for a particular user
 .get((request, response) => {
-    if(request.user) {
-        campaignPromise = campaignController.findCampaignsByUser(request.user.id);
-        campaignPromise.then(function(result) {
-            response.send(result);
-        });
+    if (request.user) {
+        Campaign.find().byUser(request.user.id)
```

Comment:
Yeah, we could do that. We'd have to create static methods rather than use what mongoose calls "query helpers", but I'm fine with that. Not sure why mongoose differentiates those out, but it seemed like the way it was set up.

Actually though, the query helpers are such lightweight wrappers around the `find()` API, I'd almost prefer scrapping them altogether and just use `find({influencer: request.user.id})`. Thoughts?